codespell_lib/data/dictionary.txt

```diff
@@ -26370,6 +26370,8 @@ spsaces->spaces
 sptintf->sprintf
 spurios->spurious
 spurrious->spurious
+sqare->square
+sqared->squared
 sqaure->square
```

Comment:
You could add -s and -d off this typo too

Reply:
80337f9